### PR TITLE
Removing betas & updating based on sunrise

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/querying/querying-the-hosted-service.mdx [hunk 1]
```diff
@@ -19,9 +19,9 @@ This query lists all the counters our mapping has created. Since we only create
 }
 ```
 
-## Using The Hosted Service
+## Using The hosted service
```
Comment (benface) [inline website/pages/en/querying/querying-the-hosted-service.mdx:22]: Haha, maybe this one shouldn't be changed, or "The" should be all lowercase too?

File: website/src/IndexPage.tsx [hunk 2]
```diff
@@ -404,8 +404,7 @@ export function SupportedNetworks() {
             )
           })}
         </ul>
-        <Text.P14 color="White64" sx={{ mt: '32px' }}>
-          *{t('index.supportedNetworks.betaWarning')}
+          <Text.P14 color="White64" sx={{ mt: '32px' }}>
         </Text.P14>
```
Comment (benface) [inline website/src/IndexPage.tsx:408]: Actually can you remove the whole `<Text.P14 color="White64" sx={{ mt: '32px' }}></Text.P14>` (now empty) component please? I would make a suggestion that you can just apply here but it seems I can't due to how the diff looks.

**Review by benface — CHANGES_REQUESTED** [review]
Two small things and good to go.

File: website/src/IndexPage.tsx (hunk 2 above)
Comment (MichaelMacaulay) [reply website/src/IndexPage.tsx:408]: Sure thign!

**Review by benface — APPROVED** [review]
LGTM 👍

**Pull request merged**